packages/limit-orders-react/src/components/CurrencyInputPanel/index.tsx

```diff
@@ -234,7 +235,7 @@ export default function CurrencyInputPanel({
     setModalOpen(false);
   }, [setModalOpen]);
 
-  const isEthereum = chainId && isEthereumChain(chainId);
+  const isEthereum = chainId && isTransactionCostDependentChain(chainId);
```

Comment:
rename `isEthereum` to `isTransactionCostDependentChain` (or other suiting name)

Reply:
also, we should be able to use this fonts instead of repeating `chainId && isTransactionCostDependentChain(chainId);` across this component